### DISCUSSION TITLE
v1.2.2 - Update SVL, low power RTC example

### DISCUSSION THREAD

**Discussion by Wenn0101** [sparkfun/Arduino_Apollo3#397] [General]
Major Updates:
Update SVL to work with Big Sur
Update RTC low power examples

Minor Updates:
Fix build problems that may occur on windows machines with spaces in username

<hr /><em>This discussion was created from the release <a href='https://github.com/sparkfun/Arduino_Apollo3/releases/tag/v1.2.2-prerelease'>v1.2.2 - Update SVL, low power RTC example</a>.</em>

**Comment by Wenn0101** [comment]
@adamgarbo Was there anything else you were hoping would make it into this release?

Can you verify that the SVL is working with your Big Sur machine?

**Comment by adamgarbo** [comment]
Thanks, @Wenn0101!

I downloaded the pre-release and it looks to compile without any issues on macOS Big Sur.

Regarding other changes, there's nothing immediately that comes to mind besides changing the default baud rate to something other than 921600. Overall, this version of the core has been very stable.

@PaulZC have there been any quirks with v1.x that you've been meaning to have addressed?

Cheers,
Adam

**Comment by adamgarbo** [comment]
Hi @Wenn0101,

Quick bug in the RTC clearInterrupt() function. You need to use `APM3_RTC` on v1.x as `Apollo3RTC` is just for v2.x.

https://github.com/sparkfun/Arduino_Apollo3/blob/9f309a09712637a58e334981d000e1ff78aa575e/libraries/RTC/src/RTC.cpp#L216

Happy to create an issue if needed.

Cheers,
Adam

**Reply by PaulZC** [reply]
Hi @Wenn0101 / @adamgarbo ,

I'm seeing the same thing... (And it is stopping me from migrating the OpenLog Artemis to 1.2.2...)

Cheers!
Paul

**Comment by adamgarbo** [comment]
Another macOS quirk. I'm not sure if it's just because I manually downloaded the Core from GitHub, but macOS requires that you manually grant permission to run artemis_svl. This is common for other open source applications, but I'd never seen it previously for the Apollo3 Core.

Cheers,
Adam

![Screen Shot 2021-05-22 at 6 15 34 PM](https://user-images.githubusercontent.com/22924092/119263415-f7238000-bbac-11eb-876c-f37e0bc0f496.png)
